src/library/simplified/extension/currency-code.js

```diff
@@ -65,6 +65,9 @@ else if (utag.isRailSearchResults() && b['entity.railSearch.searchResults.railLe
 else if (utag.isRailRateDetails() && b['entity.railSearch.railDetail.railLegs.0.railOfferItems.0.price.currency'] != undefined) {
     b['currencyCode'] = b['entity.railSearch.railDetail.railLegs.0.railOfferItems.0.price.currency'];
 }
+else if (utag.isRCO() || (b.utagPageName && b.utagPageName == "page.Checkout.Confirmation" && b.entity.productTypes && b.entity.productTypes.length == 1 && b.entity.productTypes[0] == "RAIL")){
```

Comment:
It seems that we are checking that is the rails confirmation twice. Do we need the right side of the or condition?

Reply:
You are correct, I updated isRCO with the same check so will remove the rt side.